db/migrations/20210930185746_goteo_footprint_impact_data.php

```diff
@@ -0,0 +1,63 @@
+<?php
+/**
+ * Migration Task class.
+ */
+class GoteoFootprintImpactData
+{
+  public function preUp()
+  {
+      // add the pre-migration code here
+  }
+
+  public function postUp()
+  {
+      // add the post-migration code here
+  }
+
+  public function preDown()
+  {
+      // add the pre-migration code here
+  }
+
+  public function postDown()
+  {
+      // add the post-migration code here
+  }
+
+  /**
+   * Return the SQL statements for the Up migration
+   *
+   * @return string The SQL string to execute for the Up migration.
+   */
+  public function getUpSQL()
+  {
+     return "
+      CREATE TABLE `footprint_impact` (
+        `footprint_id` INT(10) UNSIGNED NOT NULL,
```

Comment:
No id in this table?

Reply:
This table is like the ones we have for sdg_footprint were it only has a relation between footprint and impact_data